lib/active_record_shards/tasks.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 require 'active_record_shards'
 
-%w[db:drop db:create db:abort_if_pending_migrations db:reset db:test:purge].each do |name|
+%w(db:drop db:create db:abort_if_pending_migrations db:reset db:test:purge).each do |name|
```

Comment:
I prefer the %[] since it makes it obvious that it's an array

```
Style/PercentLiteralDelimiters:
  PreferredDelimiters:
    '%i': '[]'
    '%w': '[]'
    '%W': '[]'
```